lucupy/helpers/__init__.py

```diff
@@ -11,10 +11,17 @@
 
 
 def flatten(lst):
-    """
-    Flattens any iterable, no matter how irregular.
-    Example: flatten([1, 2, [3, 4, 5], [[6, 7], 8, [9, 10]]])
-    Deliberately left untyped to allow for maximum type usage.
+    """Flattens any iterable, no matter how irregular.
+       Deliberately left untyped to allow for maximum type usage.
+    
+    Example:
+        flatten([1, 2, [3, 4, 5], [[6, 7], 8, [9, 10]]])
+    
+    Args:
+        lst: n-dimensional array
```

Comment:
This really should be any sequence, possibly containing subsequences, so it should work on pretty much everything from lists containing sets containing maps, etc, but this is fine for now.